test/evergreen.yml

```diff
@@ -1990,6 +1990,48 @@ tasks:
           # No of processes to run in the background
           no_of_procs: 10
 
+  # The task name is ppc-zseries because this task will be used in both buildVariants
+  - name: format-stress-ppc-zseries-test
+    # Set 25 hours timeout
+    exec_timeout_secs: 90000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger with builtins"
+      - func: "format test script"
+        vars:
+          format_test_setting: ulimit -c unlimited
+          #run for 24 hours ( 24 * 60 = 1440 minutes), use default config
+          format_test_script_args: -b "SEGFAULT_SIGNALS=all catchsegv ./t" -t 1440
+
+  - name: format-wtperf-test
```

Comment:
One thing to note about all stress tests is that they were supposed to be self triggering, i.e., all steps of original Jenkins tests are repeated few times.

I wonder if we need to repeat the steps in `format-wtperf-test` few times to emulate stress testing? You are doing similar sort of repeated testing in `format-stress-smoke-test` as well.

Reply:
Just confirm if it would be beneficial to repeat the `wtperf` test with similar configs. I am not sure if `wtperf` randomly selects missing configs.

Reply:
I just confirmed with @kommiharibabu that wtperf doesn't randomly sets missing configs before executing, unlike format-test. So, it doesn't make sense to run it multiple times with similar configuration.

Reply:
Thanks for checking this.